recipe/bld.bat

```diff
@@ -0,0 +1,36 @@
+set CMAKE_CONFIG=Release
+
+mkdir build_shared_%CMAKE_CONFIG%
+pushd build_shared_%CMAKE_CONFIG%
+
+cmake -G "NMake Makefiles" ^
+      -DCMAKE_BUILD_TYPE:STRING=%CMAKE_CONFIG% ^
+      -DBUILD_SHARED_LIBS:BOOL=ON ^
+      "%SRC_DIR%"
+nmake
+ctest -V
+
+move bro.exe "%LIBRARY_BIN%\"
+move *.dll "%LIBRARY_BIN%\"
```

Comment:
It seems that the core reason of this difference, is in how run-time dependency loading mechanism is working on Windows. Dependencies loaded searches dependencies using PATH variable, dependencies paths are not hard-coded in binaries. So it makes sense to keep .dlls and .exes in the same folder added to the PATH. for example.